add doc links (#725)
elastic/search-ui: README.md

<img src="docs/search-ui/images/screenshot.png" width="600">

### Docs

> Checkout our [documentation site](https://docs.elastic.co/search-ui/overview)

Reviewer: ```suggestion
Check out our [documentation site](https://docs.elastic.co/search-ui/overview).
```

### Live Demo

> Checkout the [live demo of Search UI](https://search-ui-stable.netlify.com).

Reviewer: ```suggestion
Check out the [live demo of Search UI](https://search-ui-stable.netlify.com).
```

});
```

> _Read our [Custom Connector](https://docs.elastic.co/search-ui/connectors-api/custom-connector) to learn how to build a connector for any Search API._

Reviewer: ```suggestion
_Read our [Custom Connector Guide](https://docs.elastic.co/search-ui/connectors-api/custom-connector) to learn how to build a connector for any Search API._
```
